www/src/app/controls/course/course-student-form/course-student-form.component.ts

```diff
@@ -1,310 +0,0 @@
-import {Component, Input, OnInit} from "@angular/core";
```

Comment:
Why is this deleted exactly? Is it just unused?

Reply:
Yes. It's unused and wasn't functional anyways.